src/components/booking/partial/steps-actions.tsx

```diff
@@ -356,6 +355,58 @@ export const stepsActions = ({
     },
   };
 
+  const saveDataOnValidation = {
+    execute: async (formData: BookingFormData, boat: Boat) => {
+      setModalInfo({
+        modal: "loading",
+        message: t("loadingMessage.saving_information"),
+        error: "",
+      });
+
+      const {
+        ID_Back_Picture,
+        ID_Front_Picture,
+        ...bookingData
+      } = formData;
+
+
+      const bookingInfo = new Booking({
+        ...bookingData,
```

Comment:
Im not sure why we need this
I prefer just to see what we are going to update
 - name
  - last name
  - front
   - back
    - approved or not
    
    Only this 5-6 params